lib/omniauth/strategies/auth0.rb

```diff
@@ -100,7 +100,11 @@ def request_phase
 
       # Parse the raw user info.
       def raw_info
-        userinfo_url = options.client_options.userinfo_url
+        userinfo_url = if options.audience.present?
+          options.audience
```

Comment:
The problem with using `audience` here is that is can be set to an API, like the Management API or a custom resource server. If you add an OIDC scope then you can use the token you get with `/userinfo` but this would try to hit the resource server indicated during login.

Reply:
Just expanding on this a bit ... if you read the docs page you linked to, you'll see a way around this:

> You can also remove this `audience=[...]/userinfo` parameter altogether if your application is flagged as **OIDC-Conformant** in the **OAuth2** tab of the application's **Advanced Settings**.

Can you try that @wqsaali?

Reply:
@wqsaali Have you tried out what I suggested above? I believe that custom domains are working with this strategy based on my own testing and feedback in #71.

Reply:
@joshcanhelp Yes, I tried after removing the `audience` so it's still not working. Issue is that **/userinfo** should be called with auth0 provided domain instead with a custom domain. So can you check again?